### ISSUE TITLE
`DecodeAeson` instance for `Transaction`

### ISSUE THREAD

**Issue by ngua** [Plutonomicon/cardano-transaction-lib#575]
Part of #30. Perhaps blocked on https://github.com/mlabs-haskell/ogmios-datum-cache/pull/73, but the JSON serialization that is implemented should be pretty predictable (there's an example of full block data in the ODC readme)

**Comment by klntsky** [comment]
Not needed for CTL itself. CBORHex format is good enough for serialization
